views/pages/admin/exam-bank.pug

```diff
@@ -10,3 +16,33 @@ block body
       label This is a temporary button to rebuild the JSON file that the exam bank uses to populate its table.
       div(style="text-align:center; padding: 1em;")
         button(type="submit" style="border: 1px solid var(--mathsoc-pink); display: block; margin: auto; padding: 1em; cursor: pointer;") Rebuild the exam bank list.
+    div(id="content")
+      div(id="exam-bank-searchbar")
+        label(id="exams-search-label") Filter for your exam then find it in the list below
+        input(id="exam-list-filter" placeholder="CS 136" aria-labelledby="exams-search-label")
+      h2 Exams 
+      table(id="exams-table")
+        thead
+          tr 
+            th(class="exam-course") Course
+            th(class="exam-type") Type
+            th(class="exam-offering") Offering
+            th(class="exam-link") Exam
+            th(class="exam-link") Solutions
+            th(class="exam-actions") Actions
+            th(class="exam-hidden") Hidden (?)
+        tbody
+          tr(id="hidden-row" data-course-dept="$COURSE-DEPT$" data-course-code="$COURSE-CODE$")
+            td(class="exam-course") $COURSE-NAME$
+            td(class="exam-type") $TYPE$
+            td(class="exam-offering") $OFFERING$
+            td(class="exam-link") 
+              a(class="exam-download" href="$EXAM-FILE$") View exam
+            td(class="exam-link") 
+              a(class="solution-download" href="$SOLUTION-FILE$") View solutions
+            td(class="exam-actions")
```

Comment:
We should plan for the need to hide/delete only the exam file or only the solutions file - this would only provide the option to hide/delete both at once.  Can we update the table to allow for hiding/deleting each individually?